lib/msf/core/post/windows/shadowcopy.rb

```diff
@@ -182,6 +182,25 @@ def start_vss
         return false
       end
     end
+    unless start_swprv
+      return false
+    end
+    return true
+  end
+
+  def start_swprv
+    vss_state = wmic_query('Service where(name="swprv") get state')
```

Comment:
It'll be a single request using the EXTAPI query call (ignoring loading if not already there). 

Msf::Post:: should be the preferred method, so modules are uniform, more compatible across session types (Shell/PowerShell/Meterpreter), where possible. Abstracts the module away from the underlying call type. E.g. `services_info` could be extended to do a command call using WMI/PowerShell for additional future compatibility etc.